openmsupply/open-msupply
# 1948 add prescription type to invoice

Fixes #1948 

# 👩🏻‍💻 What does this PR do? 
Adds `prescription` to invoice type and translates over dispensary invoices. This will make it easier to load dispensary invoices by number. Please ignore the branch name 😓 

# 🧪 How has/should this change been tested? 
- Make sure tests still run in backend
- Frontend change:
   - Make sure store is a dispensary
   - Create some prescriptions in OG
   - Move dispensary to remote site (in OG)
   - Sync site in remote
   - Go to Outbound Shipment
   - Shouldn't be able to see Dispensary invoices
